client/Makefile

```diff
@@ -1,6 +1,5 @@
 include ../common.mk
 
-CXG_CONFIG := $(if $(CXG_CONFIG),$(CXG_CONFIG),./__tests__/e2e/test_config.yaml)
```

Comment:
Smoke shard now hardcodes the standard test config and forces the Chromium project; callout for anyone adding new projects that they should update this list too.